[Bug]: ne build failure

### Problem description

139 commands processed.
rm -f ne.info*
perl info2src.pl
Can't read command names from 'ne.info'. at info2src.pl line 64.
make[1]: *** [makefile:149: names.c] Error 2
info2src.pl: HASH_TABLE_SIZE = 3361
perl info2src.pl
Can't read command names from 'ne.info'. at info2src.pl line 64.
make[1]: *** [makefile:149: names.h] Error 2
139 commands processed.
rm -f ne.info*
perl info2src.pl
Can't read command names from 'ne.info'. at info2src.pl line 64.
info2src.pl: HASH_TABLE_SIZE = 3361
139 commands processed.
make[1]: *** [makefile:149: help.c] Error 2
info2src.pl: HASH_TABLE_SIZE = 3361
rm -f ne.info*
info2src.pl: HASH_TABLE_SIZE = 3361
139 commands processed.
139 commands processed.
rm -f ne.info*
rm -f ne.info*
make[1]: Leaving directory '/home/builder/.termux-build/ne/src/src'
llvm-strip: error: 'ne': No such file or directory
make: *** [makefile:30: build] Error 1


[ne.txt](https://github.com/termux/termux-packages/files/9693115/ne.txt)


### What steps will reproduce the bug?

build

### What is the expected behavior?

_No response_

### System information

termux-info:

```

```

--- a/packages/ne/build.sh
+++ b/packages/ne/build.sh
@@ -9,6 +9,7 @@ TERMUX_PKG_DEPENDS="libandroid-support, ncurses"
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_UPDATE_TAG_TYPE="newest-tag"
+TERMUX_MAKE_PROCESSES=1
 
 termux_step_pre_configure() {
 	export OPTS="$CFLAGS $CPPFLAGS"

